docs/docusaurus/credentials/issue.md

```diff
@@ -12,7 +12,7 @@ In the Issue Credentials Protocol, there are two roles:
 1. The [Issuer](/docs/concepts/glossary#issuer) is responsible for creating a new credential offer, sending it to a Holder, and issuing the VC once the offer is accepted.
 2. The [Holder](/docs/concepts/glossary#holder) is responsible for accepting a credential offer from an issuer and receiving the VC.
 
-The Issuer and Holder interact with the PRISM Agent API to perform the operations defined in the protocol.
+The Issuer and Holder interact with the Identus Cloud Agent API to perform the operations defined in the protocol.
```

Comment:
This is not correct. It should be "Cloud Agent" instead of "Identus Cloud Agent"